frontend/src/common/ReviewWithComment.tsx

```diff
@@ -135,6 +144,7 @@ export default function ReviewWithComment({
               value={reviewRequest}
               getOptionLabel={(option) => getRoleDisplay(option.role, modelRoles)}
               options={reviews}
+              defaultValue={typeof role === 'string' ? reviews.find((review) => review.role === role) : null}
```

Comment:
We should be setting the default value of the `reviewRequest` state variable instead of doing this. I've shown how to do this in another comment.